libs/transactions-journey/e2e-tests/specs/transaction-details.spec.ts

```diff
@@ -0,0 +1,32 @@
+import { expect, TestType } from '@playwright/test';
+import { TRANSACTION_DETAILS } from '../data/transaction-details.data';
+import { TransactionDetailsPage } from '../page-object/transaction-details.page';
+
+export interface TransactionDetailsFixture {
+  detailsPage: TransactionDetailsPage;
+  detailsData?: typeof TRANSACTION_DETAILS;
+}
+
+export function testTransactionDetails(
+  test: TestType<TransactionDetailsFixture, TransactionDetailsFixture>
+) {
+  test.describe('Transactions', () => {
```

Comment:
What's the purpose of having two describe blocks? I think we can get rid of this one.

Reply:
The idea of 2 blocks is to see the ownership of the journeys' tests.
For example, each journey can have a several screens and it is also possible that screens in 2 journeys have the same name/functionality:
```
Transactions > Transactions List
Transactions > Transactions Details
Accounts & Transactions > Transactions Details
Accounts & Transactions > Account Details
```
What do you think, is it something that might be useful for QAs or maybe you'd prefer to use tags instead or it's not needed at all ?

Reply:
Also I think we have to maintain the sequence of product catalog. In order to achieve that, we used 2 describe blocks.

Reply:
The folder structure already indicates the ownership, and with the addition of tags this would also state the ownership, admittedly it might be less clear, but it looks like having a top level describe block that just says `Transactions` seems a bit odd. 
There aren't any best practices for multiple describe blocks, except for running in parallel. I'm not against this, just want to make sure we are not causing more bloat than is necessary.

Reply:
Updated